docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1108,99 +1138,100 @@ Enables [pruning](../../Concepts/Pruning.md) to reduce storage required for the
-```
+    ```bash tab="Syntax"
+    --remote-connections-limit-enabled[=<true|false>]
+    ```
 
-```bash tab="Environment Variable"
-REVERT_REASON_ENABLED=true
-```
+    ```bash tab="Command Line"
+    --remote-connections-limit-enabled=false
+    ```
 
-```bash tab="Configuration File"
-revert-reason-enabled=true
-```
+    ```bash tab="Environment Variable"
+    BESU_REMOTE_CONNECTIONS_LIMIT_ENABLED=false
+    ```
 
-Enables including the [revert reason](../../HowTo/Send-Transactions/Revert-Reason.md) in the transaction 
-receipt. Default is `false`. 
+    ```bash tab="Configuration File"
+    remote-connections-limit-enabled=false
+    ```
 
-!!! caution 
-    Enabling revert reason may use a significant amount of memory. We do not recommend enabling revert
-    reason when connected to public Ethereum networks. 
+    Specify to limit the percentage of remote P2P connections initiated by peers. Default is true. 
 
-### remote-connections-limit-enabled
+    !!! tip
+        In private networks with a level of trust between peers, disabling the remote connection limits 
+        may increase the speed at which nodes can join the network.
 
-```bash tab="Syntax"
---remote-connections-limit-enabled[=<true|false>]
-```
+    !!! important
+        To prevent eclipse attacks, ensure the remote connections limit is enabled when connecting to 
+        any public network and especially when using [fast sync](#fast-sync-options). 
 
-```bash tab="Command Line"
---remote-connections-limit-enabled=false
-```
+    ### remote-connections-max-percentage
 
-```bash tab="Environment Variable"
-BESU_REMOTE_CONNECTIONS_LIMIT_ENABLED=false
-```
+    ```bash tab="Syntax"
+    --remote-connections-max-percentage=<DOUBLE>
+    ```
 
-```bash tab="Configuration File"
-remote-connections-limit-enabled=false
-```
+    ```bash tab="Command Line"
+    --remote-connections-max-percentage=25
+    ```
 
-Specify to limit the percentage of remote P2P connections initiated by peers. Default is true. 
+    ```bash tab="Environment Variable"
+    BESU_REMOTE_CONNECTIONS_MAX_PERCENTAGE=25
+    ```
 
-!!! tip
-    In private networks with a level of trust between peers, disabling the remote connection limits 
-    may increase the speed at which nodes can join the network.
+    ```bash tab="Configuration File"
+    remote-connections-max-percentage=25
+    ```
 
-!!! important
-    To prevent eclipse attacks, ensure the remote connections limit is enabled when connecting to 
-    any public network and especially when using [fast sync](#fast-sync-options). 
+    Percentage of remote P2P connections that can be established with the node. Must be between 0 and 100 inclusive.
+    Default is 60. 
 
-### remote-connections-max-percentage
+    ### required-block
 
-```bash tab="Syntax"
---remote-connections-max-percentage=<DOUBLE>
-```
-
-```bash tab="Command Line"
---remote-connections-max-percentage=25
-```
+    ```bash tab="Syntax"
+    --required-block=<BLOCK=HASH>
```

Comment:
I know this change wasn't part of this ticket but can `BLOCK=HASH` be updated to `BLOCK-HASH`

Reply:
The correct syntax format requires the equals sign. I changed the format to better reflect this.

Reply:
I get it now.  I looked at the CLI help - you can specify multiple blocks and hashes.  This needs to be reflected in the doc as well: 

`--required-block, --required-blocks[=BLOCK=HASH[,BLOCK=HASH...]...]`

Reply:
Updated.